content/applications/inventory_and_mrp/inventory/management/misc/batch_transfers.rst

```diff
@@ -1,124 +1,223 @@
-    :align: center
-    :alt: How to handle batch transfers with unavailable products. Creation of a backorder inside
-          of a batch transfer
+Another method of creating batch transfers is available using the :guilabel:`Add to batch` option in
+a list. Navigate to the :menuselection:`Inventory app --> Operations` drop-down menu and select any
+of the :guilabel:`Transfers` to open a filtered list of transfers.
 
-.. image:: batch_transfers/batch_transfers_10.png
-    :align: center
-    :alt: View of how backorders are handled in Odoo's batch transfers
+.. image:: batch_transfers/transfers-drop-down.png
+   :align: center
+   :alt: Show all transfer types in a drop-down menu: Receipts, Deliveries, Internal Transfers,
+         Manufacturings, Batch Transfers, Dropships.
 
-Process a Batch Transfer from the Barcode app
-=============================================
+On the transfers list, select the checkbox to the left of the selected transfers to add in a batch.
+Next, navigate to the :guilabel:`Actions ⚙️ (gear)` button and click :guilabel:`Add to batch` from
+the resulting drop-down menu.
 
-Enter the *Barcode* application, select the *Batch Transfers* menu.
+.. image:: batch_transfers/add-to-batch.png
+   :align: center
+   :alt: Use *Add to batch* button, from the *Action* button's list.
 
-.. image:: batch_transfers/batch_transfers_11.png
-    :align: center
-    :alt: View of the Odoo Barcode app dashboard
+Doing so opens an :guilabel:`Add to batch` pop-up, where the employee :guilabel:`Responsible` for
+the picking can be assigned. Choose from the two radio options to add to :guilabel:`an existing
+batch transfer` or create :guilabel:`a new batch transfer`. To begin with a draft, select the
+:guilabel:`Draft` checkbox. Conclude the process by clicking :guilabel:`Confirm`.
 
-Then, you can enter the batch transfer on which you want to work. Batch
-transfers can easily be grouped per responsible if necessary.
+.. image:: batch_transfers/add-to-batch-window.png
+   :align: center
+   :alt: Show *Add to batch* window to create a batch transfer.
 
-.. image:: batch_transfers/batch_transfers_12.png
-    :align: center
-    :alt: View of the batch transfers dashboard inside of the Barcode app
+Process batch transfer
+======================
 
-In the batch transfer, products are classified per
-location. The source document is visible on each line and a color-code
-helps differentiate them.
+Handle batch transfers in the :menuselection:`Inventory app --> Operations --> Batch Transfers`
+page. Select the intended transfer from the list, and on the batch transfer form, input the
+:guilabel:`Done` quantities for each product under the :guilabel:`Detailed Operations` tab. Finally,
+select :guilabel:`Validate` to complete the picking.
 
-.. image:: batch_transfers/batch_transfers_13.png
-    :align: center
-    :alt: View of an in progress batch transfer with the Odoo Barcode application
+.. tip::
```

Comment:
```suggestion

```

Reply:
moved it below, but can bring the whole tip up to start at line 135 -- but there's no need for the blank line between the tip RST and the tip, itself.